### DISCUSSION TITLE
How to type check the project at once

### DISCUSSION THREAD

**Discussion by vnugent** [vercel/next.js#37097] [Help] [answered]
My project uses Typescript and ts-standard for coding style. 
 When I run `yarn build`, I only see one TS error message at a time.  I have to fix it, then run `yarn build` again to see the next error (if any).  Is it possible to ask next to check the whole project and show all errors at once?

version: 12.1.7-canary.10

**Comment by GabenGar** [comment, marked as answer]
Is this something Typescript can do in the first place? Chances are the error makes the code after it invalid anyway and all you'll get is 1000+ `SyntaxError`s. 
But there is a solution with the similar question: https://github.com/vercel/next.js/discussions/17300#discussioncomment-1600974
